src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py

```diff
@@ -231,6 +232,11 @@ def _get_lexical_search_term(marqo_query: MarqoLexicalQuery) -> str:
 
             return f'{or_terms}{and_terms}'
 
+        if marqo_query.score_modifiers is not None:
```

Comment:
`VespaIndex` classes do support score modifiers for lexical search. We can do this validation somewhere else (closer to API), but also we could just leave it out for the time being.